public/index.html

```diff
@@ -6,6 +6,10 @@
   integrity="sha512-puBpdR0798OZvTTbP4A8Ix/l+A4dHDD0DGqYW6RQ+9jxkRFclaxxQb/SJAWZfWAkuyeQUytO7+7N4QKrDh+drA=="
   crossorigin=""/>
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+  <link
+  rel="icon"
+  href="https://www.shareicon.net/data/2016/06/18/596437_pig_512x512.png"
+/>
     <meta
```

Comment:
Yay! i don't know why, but it bothers me so much when the favicon isn't setup. Thanks!